ios/FluentUI/Vnext/Button/MSFButton.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Combine
+import SwiftUI
+import UIKit
+
+/// UIKit wrapper that exposes the SwiftUI Button implementation
+@objc open class MSFButton: NSObject,
+                            FluentUIWindowProvider,
+                            UIGestureRecognizerDelegate {
+
+    @objc open var action: ((_ sender: MSFButton) -> Void)?
+
+    @objc open var state: MSFButtonState {
+        return self.buttonView.state
+    }
+
+    @objc open var view: UIView {
+        return hostingController.view
+    }
+
+    @objc public convenience init(style: MSFButtonStyle = .secondary,
+                                  size: MSFButtonSize = .large,
+                                  action: ((_ sender: MSFButton) -> Void)?) {
+        self.init(style: style,
+                  size: size,
+                  action: action,
+                  theme: nil)
```

Comment:
While we're here, can we consolidate these `init` functions? The designated initializer already has a default `nil` theme, so I think this `convenience init` can just be removed.

Reply:
The problem with just setting the default to nil in objc exposed method signatures is that the parameter is required, which makes it more verbose to call in objc.